packages/neutron-sdk/src/interchain_queries/v045/types.rs

```diff
@@ -403,3 +470,61 @@ impl KVReconstruct for Delegations {
         Ok(Delegations { delegations })
     }
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+pub struct UnbondingEntry {
+    pub balance: Uint128,
+    pub completion_time: Option<Timestamp>,
+    pub creation_height: u64,
+    pub initial_balance: Uint128,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+pub struct UnbondingResponse {
+    pub delegator_address: Addr,
+    pub validator_address: String,
+    pub entries: Vec<UnbondingEntry>,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+/// A structure that can be reconstructed from **StorageValues**'s for the
+/// **Delegator Unbonding Delegation Interchain Query**.
+/// Contains unbonding delegations which some delegator has on remote chain.
+pub struct UnbondingDelegations {
+    pub unbonding_responses: Vec<UnbondingResponse>,
+}
+
+impl KVReconstruct for UnbondingDelegations {
+    fn reconstruct(storage_values: &[StorageValue]) -> NeutronResult<UnbondingDelegations> {
+        let mut unbonding_responses: Vec<UnbondingResponse> =
+            Vec::with_capacity(storage_values.len());
+
+        for storage_value in storage_values {
+            let unbonding_delegation_sdk: UnbondingDelegation =
+                UnbondingDelegation::decode(storage_value.value.as_slice())?;
```

Comment:
It is necessary to use key, too, because if there is no value, delegator_address and validator_address will be empty.

Reply:
Is it request or comment?

Reply:
and this is your code, isn't it?

Reply:
oops, this was a comment I left for myself. I will fix it right now, sorry!

Reply:
Turns out it is impossible to do since converting address to storage key loses data (in this key, it loses address prefix).

I will simply not do it then :)